code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -627,14 +627,15 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/screw = 0 // kinky
 	var/super = 0 //for the fattest vapes dude.
 	var/emagged = 0 //LET THE GRIEF BEGIN
+	var/param_color

```

Comment:
It was a parameter for new. It's being stuck there so that vapes will continue to work without other changes.